storm-core/src/jvm/org/apache/storm/command/Heartbeats.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.storm.command;
+
+import com.google.common.base.Joiner;
+import org.apache.storm.cluster.ClusterStateContext;
+import org.apache.storm.cluster.ClusterUtils;
+import org.apache.storm.cluster.IStateStorage;
+import org.apache.storm.generated.ClusterWorkerHeartbeat;
+import org.apache.storm.stats.StatsUtil;
+import org.apache.storm.utils.ConfigUtils;
+import org.apache.storm.utils.Utils;
+import org.json.simple.JSONValue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+public class Heartbeats {
+    private static final Logger LOG = LoggerFactory.getLogger(Heartbeats.class);
+
+    public static void main(String[] args) throws Exception {
+        String command = args[0];
```

Comment:
should we have a check for length here? ArrayOutOfBoundsException may be confusing for cli user.